membrane_rtc_engine/lib/membrane_rtc_engine/engine.ex

```diff
@@ -520,6 +520,22 @@ defmodule Membrane.RTC.Engine do
     {actions, state}
   end
 
+  @impl true
+  def handle_child_pad_removed(
+        {:endpoint, _id},
+        {Membrane.Pad, :output, {track_id, _variant}},
+        ctx,
+        state
+      ) do
+    track_tee = get_track_tee(track_id, ctx)
```

Comment:
This way, other endpoints will get hanlde_pad_removed without getting tracks_removed at first. I think we should more look at `handle_child_notification(:removed_tracks)` in the webrtc endpoint and instead of removing track_senders, just publish `removed_tracks` and remove tracks senders in handle_pad_removed :thinking: ?